src/components/AntiPatternActionsComponent.vue

```diff
@@ -28,5 +28,19 @@
     @Component
     export default class AntiPatterns extends Vue {
         @Prop(Object) public antiPattern!: AntiPattern;
+        public shareLink: string = '';
```

Comment:
I think instead of initializing these variables with empty strings it would be 'cleaner' to use computed props like this
```java
public get shareLink() {
...
}
```